subworkflows/local/bam_variant_calling_somatic_all/main.nf

```diff
@@ -176,7 +176,7 @@ workflow BAM_VARIANT_CALLING_SOMATIC_ALL {
 
     // MSISENSOR
     if (tools.split(',').contains('msisensorpro')) {
-        MSISENSORPRO_MSI_SOMATIC(cram.combine(intervals_bed_combined), fasta, msisensorpro_scan, intervals_bed_combined)
+        MSISENSORPRO_MSI_SOMATIC(cram.combine(intervals_bed_combined), fasta, msisensorpro_scan)
```

Comment:
I don't understand how the tests even passed 🙈

Reply:
I'm confused as well...